Libraries/LibM/math.cpp

```diff
@@ -474,4 +474,66 @@ double modf(double x, double* intpart)
     *intpart = (double)((int)(x));
     return x - (int)x;
 }
+
+double gamma(double x)
+{
+    // Stirling approximation
+    return sqrt(2.0 * M_PI / x) * pow(x / M_E, x);
+}
+
+double expm1(double x)
+{
+    return pow(M_E, x) - 1;
+}
+
+double cbrt(double x)
+{
+    if (x > 0) {
+        return pow(x, 1 / 3);
```

Comment:
This can't be right. `1` and `3` are integers here, so you'll be calling `pow(x, 0)`

You can expose these functions on the `Math` object in `LibJS`. That will allow you to add tests for them so we can see that they work :)

Reply:
Good catch!

I've tried adding the tests in LibJS, but they all fail, even though the output is correct in the console.. 🤔 

![image](https://user-images.githubusercontent.com/940154/85222721-d14e9380-b3bd-11ea-8cfc-9086c37869a4.png)

Reply:
Yeah I think this happens because string/double conversion is not super great in Serenity (yet).
You can work around the issue by using the `isClose()` helper, see for example how it's done in the `Math.exp.js` test :)

Reply:
Unfortunately `isClose` isnt close enough in this case; 
Chrome gives the value
`Math.acosh(2) == 1.3169578969248166`
while serenity gives 
`Math.acosh(2) == 1.3169589711848481`

And that falls just outside the isClose range :(
Its the same, if not worse for `asinh` and `atanh`. So either `exp` or `log` (or both) needs an update aswell